backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt

```diff
@@ -37,20 +37,34 @@ class PatientService(
     /**
      * Returns single patient with given personal number or throws exception.
      */
-    suspend fun getPatientByPersonalNumber(patientPersonalNumber: String): PatientDtoOut =
+    private suspend fun getPatientByPersonalNumber(patientPersonalNumber: String): PatientDtoOut? =
         patientRepository.getAndMapPatientsBy {
             Patients.personalNumber eq patientPersonalNumber.normalizePersonalNumber()
         }.singleOrNull()?.withSortedAnswers()
-            ?: throw entityNotFound<Patients>(Patients::personalNumber, patientPersonalNumber)
 
     /**
      * Returns single patient with given insurance number or throws exception.
      */
-    suspend fun getPatientByInsuranceNumber(patientInsuranceNumber: String): PatientDtoOut =
+    private suspend fun getPatientByInsuranceNumber(patientInsuranceNumber: String): PatientDtoOut? =
```

Comment:
roc? Nullable je fuj ~ cesta do pekel.

Reply:
hele kdyz je to private tak cajk, navic tady to dava smysl